crates/wikipedia-dict/assets/dictionary.txt

```diff
@@ -3477,7 +3477,6 @@ seceeded->succeeded, seceded
 secratary->secretary
 secretery->secretary
 sedereal->sidereal
-seeked->sought
```

Comment:
This should be reverted.  We only use `typos-dict`.  The others are for comparison purposes